src/test/scala/fi/oph/koski/meta/GithubActionsSpec.scala

```diff
@@ -17,6 +17,7 @@ class GithubActionsSpec extends AnyFreeSpec with Matchers {
       .filterNot(_.contains("fi.oph.koski.valpasfront")) // Valpas frontend
       .filterNot(_.contains("fi.oph.koski.localization")) // Testit rikki
       .filterNot(_.contains("fi.oph.koski.casintegration")) // Testit rikki, vaatii setuppia?
+      .filterNot(_.contains("fi.oph.koski.integrationtest")) // Testit rikki, vaatii setuppia?
```

Comment:
saisikohan nämä mukaan run_smoketests.yml:iin? Vaikuttavat jotenkin sen luonteisilta.

Reply:
Näitä ei oo ylläpidetty vuosiin niin poistan